website/content/docs/agent/config/config-files.mdx

```diff
@@ -542,7 +542,7 @@ Refer to the [formatting specification](https://golang.org/pkg/time/#ParseDurati
 
 - `license_path` <EnterpriseAlert inline /> This specifies the path to a file that contains the Consul Enterprise license. Alternatively the license may also be specified in either the `CONSUL_LICENSE` or `CONSUL_LICENSE_PATH` environment variables. See the [licensing documentation](/consul/docs/enterprise/license/overview) for more information about Consul Enterprise license management. Added in versions 1.10.0, 1.9.7 and 1.8.13. Prior to version 1.10.0 the value may be set for all agents to facilitate forwards compatibility with 1.10 but will only actually be used by client agents.
 
-- `limits`: This block specifies various types of limits that the Consul server agent enforces.
+- `limits`: This block specifies various types of limits that the Consul server agent enforces. Declaration differs between Consul Agent Server and Consul Agent Client. For Server, use `limits { ... }` and for Client, use `limits = { ... }`. Not adding the `=` sign for the stanza in the Client config could show the error `expected a map, got slice` upon reloading or restarting the service.
```

Comment:
```suggestion
- `limits`: This block specifies various types of limits that the Consul server agent enforces. When specifying limits for Consul server agents, use `limits { ... }`. When specifying limits for Consul client agents, use `limits = { ... }`. If the `=` character is excluded from client configurations, Consul prints the following error message when reloading or restarting the service: `expected a map, got slice` .
```
A few suggestions for consistency with our style guide. Also, I think the `limits` block is only intended to be specified in server agent configurations and not in clients.  I need to check with the engineering team, but we may need to change this.

Reply:
```suggestion
- `limits`: This block specifies various types of limits that the Consul server agent enforces. Configuring limits does not affect the behavior of Consul agents in client mode. 
```
I verified with the product team that limit configurations do not affect clients. This suggestion makes it explicit that you should only configure these settings for server.